### PR TITLE
Adding functionality to the dashboard (the new backoffice)

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update 'Adding functionality to the dashboard':

- [x] css to CSS
- [x] shorter sentence 

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)

The new backoffice

### CODE REVIEW

File: new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This page is a work in progress. It will be updated as the software evolves.
 
 ## Overview
 
-This is step 2 in our guide to building a Custom Dashboard. This step continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md), but goes further to show how to add functionality and data to our dashboard.
+This is step 2 in our guide to building a Custom Dashboard. This step continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md). We will also demonstrate how to add functionality and data.
```
Comment (alina-tincas) [inline new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md:9]: Currently, it seems there are 3 very short sentences, and if read out loud might sound a bit robotic. How about leaving the 2nd sentence together like this? 🙈 
```suggestion
This is step 2 in our guide to building a Custom Dashboard. It continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md) and will demonstrate how to add functionality and data.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
HI @erikjanwestendorp thank you for the PR 🙌 

I have a small suggestion, can you look at it and let me know what you think? 😊

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for your review. Suggestion seems good to me, just updated the PR. 😄

**Comment by alina-tincas** [comment]
Awesome, all looks good now so am merging it in 🎉 🙌

**Pull request merged**